cluster-service/Makefile

```diff
@@ -2,23 +2,29 @@ SHELL = /bin/bash
 
 RESOURCEGROUP ?= aro-hcp-${AKSCONFIG}-$(USER)
 REGION ?= $(shell az group show -n ${RESOURCEGROUP} --query location -o tsv)
+IMAGE_TAG ?= 1720806765
 
 deploy:
 	DOMAIN=$(shell az network dns zone list -g ${RESOURCEGROUP} --query "[?zoneType=='Public'].name" -o tsv) && \
 	sed -e "s/DOMAIN/$${DOMAIN}/g" -e "s/REGION/${REGION}/g" deploy/mvp-provisioning-shards.yml > deploy/tmp-provisioning-shard.yml
 	oc process --local -f deploy/openshift-templates/arohcp-namespace-template.yml \
 	  -p ISTIO_VERSION=asm-1-20 | oc apply -f -
-	oc process --local -f deploy/openshift-templates/arohcp-db-template.yml | oc apply -f -
+
 	oc process --local -f deploy/openshift-templates/arohcp-secrets-template.yml \
-	  -p PROVISION_SHARDS_CONFIG="$$( base64 -i deploy/tmp-provisioning-shard.yml)" | oc apply -f -
+		-p DB_HOST=$(shell az postgres flexible-server list --resource-group ${RESOURCEGROUP} --query "[?starts_with(name, 'cs-pg-')].fullyQualifiedDomainName" -o tsv) \
+		-p DB_NAME=cluster-service \
+		-p DB_USER=cs-pg-aaaaaaaaaaaaa-db-admin-msi \
```

Comment:
this is the name I see in Az portal under Entra admins.  This db username yields the error: 

` Can't connect to the database: pq: Service Principals cannot generate AAD_AUTH_TOKENTYPE_APP_USER tokens for role`

any other name (such as `clusters-service-eastus` or `cs-pg-aaaaaaaaaaaaa` (db name)) will timeout.

When I added my own user (my email at redhat) as Entra admin to this db instance, I can successfully login.

Reply:
Thanks for figuring this out. I guess the MI backed account lacks some permissions. I will have a look